src/snowcli/cli/streamlit/manager.py

```diff
@@ -99,13 +99,14 @@ def deploy(
             f"call SYSTEM$GENERATE_STREAMLIT_URL_FROM_NAME('{streamlit_name}')"
         )
         base_url = query_result.fetchone()[0]
-
-        url = self._get_url(base_url, qualified_name)
-
-        if open_in_browser:
-            typer.launch(url)
-        else:
-            log.info(url)
+        # TODO fix it!
+        return base_url
+        # url = self._get_url(base_url, qualified_name)
+        #
+        # if open_in_browser:
+        #     typer.launch(url)
+        # else:
+        #     return url
```

Comment:
I have no idea what to do here, `_get_url` throws error because in our host are only 3 parts

Reply:
Let's fix it first before merging this PR. Wh we have this validation? Does it has to be always 4 part?